torchaudio/_internal/module_utils.py

```diff
@@ -123,3 +123,11 @@ def wrapped(*args, **kwargs):
                 raise RuntimeError(f'{func.__module__}.{func.__name__} requires sox')
             return wrapped
     return decorator
+
+
+def is_ctc_decoder_available():
```

Comment:
I have a mixed feeling about having this function inside of `torchaudio`.

- Importing `torchaudio.prototype.ctc_decoder` might cause an unrecoverable effect like segfault
- Having this function here might encourage the use in library or other place, which goes against the purpose of logically separating the initialization of `torchaudio.prototype.ctc_decoder`, giving users control when that module should be initialized, for the fear of unrecoverable error.
- Yet, I am somewhat confident that's such error would not happen.

It's probably safer to move this inside of test module. What do you think?